smart-contracts/contracts-common/concordium-contracts-common/src/schema.rs

```diff
@@ -774,7 +774,7 @@ impl Deserial for FunctionV2 {
         if matches!(idx, 1 | 2 | 5 | 6) {
             let _ = r.return_value.insert(source.get()?);
         }
-        if matches!(idx, 3 | 4 | 5 | 6) {
+        if matches!(idx, 3..=6) {
```

Comment:
Why is this changed?

I don't think this is warranted and does not increase readability.

Reply:
Clippy complained about it, otherwise I wouldn't have changed it.
https://github.com/Concordium/concordium-base/actions/runs/8157713192/job/22298013915